EduID/Flows/PersonalInfo/EmailEditor/EmailEditorViewController.swift

```diff
@@ -50,29 +50,29 @@ class EmailEditorViewController: UIViewController, ScreenWithScreenType {
         setupUI(isLoading: true)
         guard let url = notification.userInfo?[Constants.UserInfoKey.emailUpdateUrl] as? URL else {
             assertionFailure("Email update URL not found in notification object!")
-            onUpdateFailed()
+            onUpdateFailed(nil)
             return
         }
         Task {
             do {
                 if let result = try await viewModel.confirmEmailUpdate(url: url) {
                     delegate?.goBackToInfoScreen(updateData: true)
                 } else {
-                    onUpdateFailed()
+                    onUpdateFailed(nil)
```

Comment:
wouldn’t a default parameter be more efficient? you could omit passing nil.

Reply:
No, because then the developer might forget it.
We should pass an error here when possible, because then there will be more debug info